lis/surfacemodels/land/noahmp.4.0.1/da_snow_simple/noahmp401_snowsimple_update.F90

```diff
@@ -0,0 +1,270 @@
+!-----------------------BEGIN NOTICE -- DO NOT EDIT-----------------------
+! NASA Goddard Space Flight Center
+! Land Information System Framework (LISF)
+! Version 7.3
+!
+! Copyright (c) 2020 United States Government as represented by the
+! Administrator of the National Aeronautics and Space Administration.
+! All Rights Reserved.
+!-------------------------END NOTICE -- DO NOT EDIT-----------------------
+!BOP
+! !ROUTINE: noahmp401_snowsimple_update
+! \label{noahmp401_snowsimple_update}
+!
+! !REVISION HISTORY:
+!  22 Nov 2022: Isis Brangers; Adjusted update from da_snow
+!  This version divides updates over layers and compared to da_snow 
+!  is simplified by removing non update related model routines
+! !INTERFACE:
+subroutine noahmp401_snowsimple_update(n, t, dsneqv, dsnowh)
+
+  use LIS_coreMod
+  use NoahMP401_lsmMod
+  use module_sf_noahmplsm_401
+  use noahmp_tables_401
+
+  implicit none
+! 
+! !DESCRIPTION: 
+!  This subroutine updates relevant snow prognostics based
+!  on the update to the total SWE (dsneqv) and total
+!  snow depth (dsnowh). The updated variables include
+!  number of snow layers, snice, snliq, snow temperature 
+!  and snow thickness. 
+! 
```

Comment:
Add something like: "The 'simple' update trusts that the Kalman filter will properly determine dsneqv and dsnowh, and will therefore not further partially postprocess or compact snow afterwards (unlike the default snow_update routine). The model will deal with any further processing at the next time step."

Reply:
Thx! Made corresponding changes.